core/market/forwarding/src/api.rs

```diff
@@ -8,8 +8,16 @@ use ya_client::model::NodeId;
 use ya_persistence::executor::DbExecutor;
 use ya_service_api_web::middleware::Identity;
 
+pub const CENTRAL_MARKET_URL_ENV_VAR: &str = "YAGNA_API_URL";
```

Comment:
i think the env var should be `YAGNA_MARKET_URL`, since `YAGNA_API_URL` is missing the proper postfix

source: https://github.com/golemfactory/yagna/blob/master/.env-template#L21

Reply:
of course
I had distractors writing it...